cds_ils/importer/providers/cds/rules/values_mapping.py

```diff
@@ -40,6 +40,1273 @@
     "migration": ["migration"],
 }
 
+ACQUISITION_METHOD = {
+    # possible user types (created_by.type/created_by.value)
+    "user": ["H", "R"],
```

Comment:
what is the meaning of these letters? Can we have something more readable? Or are this values coming from the migrated data?

Reply:
Your comment is still valid, maybe @kprzerwa can answer better. I just want to clarify that this is duplicated code that I added wrongly and it's not part of this PR, I already removed it, sorry...